engine/src/main/java/com/sandboni/core/engine/Processor.java

```diff
@@ -180,11 +191,21 @@ private Builder getBuilder(Context context) {
         return new Builder(context);
     }
 
+    private boolean reflectionCallersFound(Context context) {
```

Comment:
I'd prefer this method will not be part of the Processor. probably a new component (something like context-scanner/analyzer) ?

Reply:
you sure? actual filtering is done through VertexFilterFactory.get(REFLECTION_SRC).filter, 
And the rest of the logic is very case-specific..

Reply:
yes. the logic itself is done in processor (return true/false) - what would happen if we'll need further decision to make - if there is reflection and condition-x and not condition-y?
I see the Processor as  flow manager that receive inputs and make a decision to which path to proceed